plugins/inputs/intel_pmu/activators_test.go

```diff
@@ -386,21 +386,21 @@ func TestActivateEventForPlacement(t *testing.T) {
 		activeEvents, err := mActivator.activateEventForPlacements(nil, mPlacements)
 		require.Error(t, err)
 		require.Contains(t, err.Error(), "core event is nil")
-		require.Nil(t, activeEvents)
+		require.Len(t, activeEvents, 0)
```

Comment:
How about `require.Empty`? Same for the other instances.

Reply:
@srebhan Makes sense. Changed. Thanks!